lib/widgets/explore/sidelayout.dart

```diff
@@ -42,24 +62,126 @@ class SideLayoutState extends State<SideLayout> {
+                          });
+                        }
+                      },
+                    );
+                  }).toList(),
+                ),
+              ),
+              Row(
+                children: const [
+                  CircleAvatar(
+                    backgroundColor: Color(0xFFFFCD78),
+                    child: Icon(Icons.trending_up_sharp,
+                        size: 30, color: Colors.black),
+                  ),
+                  Text(
+                    " Difficulty Level",
+                    style: filterHeaderStyle,
+                  ),
+                ],
+              ),
+              Expanded(
+                child: ListView(
+                  controller: ScrollController(),
+                  children: difficultyLevelValues.keys.map((String key) {
                     return CheckboxListTile(
                       contentPadding: const EdgeInsets.all(0),
                       dense: true,
                       controlAffinity: ListTileControlAffinity.leading,
                       title: Text(key, style: filterItemStyle),
-                      value: values[key],
+                      value: difficultyLevelValues[key],
                       onChanged: (bool? value) {
                         if (value != null) {
                           setState(() {
-                            values[key] = value;
+                            difficultyLevelValues[key] = value;
                           });
                         }
                       },
                     );
                   }).toList(),
                 ),
-              )
+              ),
+              Row(
+                children: const [
+                  CircleAvatar(
+                    backgroundColor: Color(0xFFFFCD78),
+                    child: Icon(Icons.fitness_center,
+                        size: 30, color: Colors.black),
+                  ),
+                  Text(
+                    " Equipment",
+                    style: filterHeaderStyle,
+                  ),
+                ],
+              ),
+              Expanded(
+                child: ListView(
+                  controller: ScrollController(),
+                  children: equipmentValues.keys.map((String key) {
+                    return CheckboxListTile(
+                      contentPadding: const EdgeInsets.all(0),
+                      dense: true,
+                      controlAffinity: ListTileControlAffinity.leading,
+                      title: Text(key, style: filterItemStyle),
+                      value: equipmentValues[key],
+                      onChanged: (bool? value) {
+                        if (value != null) {
+                          setState(() {
+                            equipmentValues[key] = value;
+                          });
+                        }
+                      },
+                    );
+                  }).toList(),
+                ),
+              ),
```

Comment:
I think if you use a ListBuilder, it might be less repetitive + fix your issue with nested scrolling. If not, maybe try and see if you can have nested `ListView` so that the parent one has a `ScrollController()` and the inner one's dont? (I tried doing this but I think I'm missing something with the `ListView`'s)

More info: https://docs.flutter.dev/cookbook/lists/mixed-list

Reply:
Made this ages ago, I believe there was some reason I used ListView over ListBuilder... can't quite remember though. Will experiment with the nested ListView though.

Reply:
May not have time to get it in for this PR if we are trying to have everything in for 730pm.